.github/workflows/onpush.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         python-ver: ['3.8', '3.9', '3.10', '3.11' ]
-        os: [ubuntu-22.04, macos-10.15]
```

Comment:
But I actually do see some runners here: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources